src/Controller/Controller.php

```diff
@@ -211,20 +211,6 @@ public function __construct(ServerRequest $request = null, Response $response =
 
         $this->initialize();
 
-        if (isset($this->components)) {
-            triggerWarning(
-                'Support for loading components using $components property is removed. ' .
-                'Use $this->loadComponent() instead in initialize().'
-            );
-        }
-
-        if (isset($this->helpers)) {
-            triggerWarning(
-                'Support for loading helpers using $helpers property is removed. ' .
-                'Use $this->viewBuilder()->setHelpers() instead.'
-            );
-        }
```

Comment:
I thought we were going to keep these to save folks time when they accidentally used the old properties.

Reply:
=> https://github.com/cakephp/cakephp/commit/d838f352401f84f501d30ae9b620b3c7314c66c2 added in 4.x

Reply:
Ah, I'll revert it.